notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -49,9 +49,7 @@
   {
```

Comment:
<p>mix_notebook: Notebook JSON that we send to browser. It's a mix of pre and post content. post_notebook: Notebook JSON after the commit.</p><p>For each cell in post_notebook, we find a corresponding cell in mix_notebook.And add post_cell_index property to that corresponding cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>